docker/ubuntu/baseimage-20.04/Dockerfile

```diff
@@ -147,6 +132,54 @@ RUN rm -rf luacov-coveralls-master
 
 # }}}
 
+# Other development dependencies / tools.
+# NOTE: libtool-bin is due to a libzmq issue, see https://github.com/zeromq/libzmq/pull/1497
+# (can be removed if libzmq is bumped or the build switched from autotools to cmake).
+ARG MISC_TOOLS="\
+    autoconf2.64 \
+    automake \
+    ccache \
+    chrpath \
```

Comment:
to be removed?

Reply:
I figured since it's so small and a useful tool it might as well be added to the base image, but I can remove it if you prefer.

Reply:
Nah, it's fine, like I said it's only a few dozen kB.